IBPSA/Fluid/Chillers/BlackBoxData/EuropeanNorm2D.mo

```diff
@@ -0,0 +1,216 @@
+within IBPSA.Fluid.Chillers.BlackBoxData;
+model EuropeanNorm2D
+  "Performance data coming from manufacturer according to European Standards"
+  extends IBPSA.Fluid.Chillers.BlackBoxData.BaseClasses.PartialChillerBlackBox(
+    datasource=dataTable.device_id,
+    mEva_flow_nominal=dataTable.mEva_flow_nominal*finalScalingFactor,
+    mCon_flow_nominal=dataTable.mCon_flow_nominal*finalScalingFactor,
+    QUseBlackBox_flow_nominal=
+        Modelica.Blocks.Tables.Internal.getTable2DValueNoDer2(
+        tableConID,
+        TCon_nominal - 273.15,
+        TEva_nominal - 273.15));
```

Comment:
Depending on the use-case (e.g. to avoid non-literal warnings), modifications should still be possible. I added final to the datasource. The other ones are non-final.